package.json

```diff
@@ -57,4 +58,4 @@
     "truffle": "^5.1.26",
     "truffle-hdwallet-provider": "^1.0.0-web3one.1"
   }
-}
+}
```

Comment:
I thought we wanted newlines at the end of the files?
```suggestion
}

```

Reply:
This was done by yarn or npm. Wasn’t me. It’s the automatic linter